daprdocs/content/en/developing-applications/debugging/debug-k8s/debug-dapr-services.md

```diff
@@ -0,0 +1,108 @@
+---
+type: docs
+title: "Debug Dapr services on Kubernetes"
+linkTitle: "Dapr services"
+weight: 10000
+description: "How to debug Dapr services on your Kubernetes cluster"
+---
+
+## Overview
+
+Sometimes it is necessary to understand what's going on in Dapr Kubernetes services, including `dapr-sidecar-injector`, `dapr-operator`, `dapr-placement`, and `dapr-sentry`, especially when you diagnose your Dapr application and wonder if there's something gets wrong in Dapr itself, or when you contribute to Dapr relevant to Kubernetes support.
+
+If this is the case, Dapr has built-in Kubernetes debug support coming in to rescue.
+
+## Debug Dapr Kubernetes services
+
+Before read on, pls. refer [this guide]({{< ref kubernetes-deploy.md >}}) to learn how to deploy Dapr to your Kubernetes cluster.
+
+### 1. Build Dapr debugging binaries
+
+In order to debug Dapr Kubernetes services, it's required to rebuild all Dapr binaries and Docker images to disable compiler optimization. To do this, pls. execute the following commands:
+
+```bash
+git clone https://github.com/dapr/dapr.git
+cd dapr
+make release GOOS=linux GOARCH=amd64 DEBUG=1
+```
+
```

Comment:
```suggestion

>On Windows download [MingGW](https://sourceforge.net/projects/mingw/files/MinGW/Extension/make/mingw32-make-3.80-3/) and use `ming32-make.exe` instead of `make`.
```